src/cursor/cur_layered.c

```diff
@@ -505,7 +518,8 @@ __clayered_next(WT_CURSOR *cursor)
     /* If we aren't positioned for a forward scan, get started. */
     if (clayered->current_cursor == NULL || !F_ISSET(clayered, WT_CLAYERED_ITERATE_NEXT)) {
         WT_ERR(__clayered_iterate_constituent(clayered, clayered->ingest_cursor, true));
-        WT_ERR(__clayered_iterate_constituent(clayered, clayered->stable_cursor, true));
+        WT_ERR_NOTFOUND_OK(
```

Comment:
Shall we still use `WT_ERR_NOTFOUND_OK`?

Reply:
Oops, I thought I fixed that - thanks. Fixed in `cb92e041c`.